### PR TITLE
Changed color of genres in anime_info page

### LINKED ISSUE #609: Change Color of genres in `anime_info` page
image

### PR DESCRIPTION
**Pull request by girish-07**
Color of genres in anime_info page is changed to primary-500. This fixes #609

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -287,7 +287,7 @@
 
                             <anime-genres class="flex gap-2 pb-2 pt-3 md:gap-[0.5vw] md:pt-[0.5vw]">
                                 {#each anime.genres as item}
-                                    <span class="rounded-lg bg-surface-900 p-2 px-3 text-xs md:rounded-[0.35vw] md:px-[0.75vw] md:py-[0.4vw] md:text-[0.75vw] md:font-semibold">{item}</span>
+                                    <span class="rounded-lg bg-primary-500 p-2 px-3 text-xs md:rounded-[0.35vw] md:px-[0.75vw] md:py-[0.4vw] md:text-[0.75vw] md:font-semibold">{item}</span>
```
[the remark this answers is not in the document]
Comment (girish-07) [reply src/routes/+page.svelte:290]: Screenshot 2023-08-20 at 6 58 10 PM
<br /><br />
Is it looking good? I feel the text is getting dimmed

[the remark this answers is not in the document]
Comment (baseplate-admin) [reply src/routes/+page.svelte:290]: Use `text-black`

[the remark this answers is not in the document]
Comment (girish-07) [reply src/routes/+page.svelte:290]: I thought of that, done its perfect
Screenshot 2023-08-20 at 7 10 21 PM

[the remark this answers is not in the document]
Comment (tokitou-san) [reply src/routes/+page.svelte]: Yea, LGTM
@baseplate-admin @girish-07 Do we need to add this in homepage > slider > genres?
or keep it `surface-900` ? whatya think?

[the remark this answers is not in the document]
Comment (baseplate-admin) [reply src/routes/+page.svelte]: Upto you. You are the primary designer

[the remark this answers is not in the document]
Comment (girish-07) [reply src/routes/+page.svelte]: I feel surface-900 is better

**Review by tokitou-san — APPROVED** [review]
Everything looks good, time to merge :smiley:

**Pull request merged**